extensions/sns/README.md

```diff
@@ -9,90 +9,40 @@ dfx sns [subcommand] [flag]
 
 Depending on the `dfx sns` subcommand you specify, additional arguments, options, and flags might apply. For reference information and examples that illustrate using `dfx sns` commands, select an appropriate command.
 
-| Command                                            | Description                                                    |
-|----------------------------------------------------|----------------------------------------------------------------|
-| [`create`](#_dfx_sns_create)                       | Creates an SNS configuration template.                         |
-| [`validate`](#_dfx_sns_validate)                   | Checks whether the sns config file is valid.                   |
-| [`deploy`](#_dfx_sns_deploy)                       | Deploys SNS canisters according to the local config.           |
-| [`prepare-canisters`](#_dfx_sns_prepare-canisters) | Prepares dapp canister(s) for SNS decentralization.            |
-| [`propose`](#_dfx_sns_propose)                     | Submits a CreateServiceNervousSystem NNS Proposal.             |
-| `help`                                             | Displays usage information message for a specified subcommand. |
+| Command                                                            | Description                                                                   |
+| ------------------------------------------------------------------ | ----------------------------------------------------------------------------- |
+| [`init-config-file validate`](#_dfx_sns_init-config-file_validate) | Checks whether the sns config file is valid.                                  |
+| [`deploy-testflight`](#_dfx_sns_deploy-testflight)                 | Creates a test deployment of the SNS canisters according to the local config. |
+| [`prepare-canisters`](#_dfx_sns_prepare-canisters)                 | Prepares dapp canister(s) for SNS decentralization.                           |
```

Comment:
This description is too abstract. Specifically, the reader is unlikely to know what "prepare" means.

IIUC, prepare-canisters is used to set the controllers of the dapp canisters.

Reply:
Done! (BTW that description was not introduced in this PR)